src/components/Product/ProductList/ProductContent/ProductRadioBox/ProductInfo/ProductInfo.scss

```
@@ -0,0 +1,75 @@
+@import '../../../../../../styles/variables.scss';
+@import '../../../../../../styles/mixin.scss';
+
+.container {
+  display: none;
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 100%;
+    height: 100%;
+    background-color: yellow;
+  }
+  &.active {
```

한 줄씩 띄워주시면 가독성이 더 좋을 것 같습니다.
```suggestion
  display: none;
  
  &::before {
    content: '';
    display: inline-block;
    width: 100%;
    height: 100%;
    background-color: yellow;
  }
  
  &.active {
```